src/components/ImportCampaign.vue

```diff
@@ -2,15 +2,13 @@
   <div class='column'>
     <div style='width: 50%'>
       <h1>Import data</h1>
-      Campaign dir : <input ref="campaignDir"><br>
-      Campaign name : <input ref="campaignName"><br>
-      Id malette : <input ref="idMalette" value="1"><br>
-      Id rederbro : <input ref="idRederbro" value="1"><br>
-      Camera number : <input ref="cameraNB" value="6"><br>
-      Description : <input ref="description" value="An amazing campaign"><br>
+      Campaign dir : <input ref='campaignDir'><br>
+      Campaign name : <input ref='campaignName'><br>
+      Id malette : <input ref='idMalette' value='1'><br>
+      Id rederbro : <input ref='idRederbro' value='1'><br>
+      Camera number : <input ref='cameraNB' value='6'><br>
+      Description : <input ref='description' value='An amazing campaign'><br>
       <input type='button' value='Launch import' v-on:click='launch()'><br>
-      Log file : <input ref="logFile">
```

Comment:
We should really have log (that the base for debugging), file or something else.